AdaSupport/embed
# INT- 669: Store chatter info on client [Safari Fix]

### Why
* Safari is no longer allowing 3rd party storage to be preserved
* https://support.apple.com/en-ca/guide/safari/sfri40732/mac
### What
* Allow for Embed to save `chatterToken` and `created` on client storage and pass it back to chat to re-initialize the chatter
* The values will be passed out `query` and in through `postMessage` 
* The pass in values will only be used if chat storage finds nothing
* https://whimsical.com/5GyVYJ9y5j3dsUdPHxz8Y9
### Testing
* Check out this and the embed branch
* Start a chat
* Clear the chat's storage
* Launch again and the same chat should reload
### Deploy
* Deploy with chat fix: https://github.com/AdaSupport/chat/pull/703
---
- [ ] Changes are CATO/WCAG compliant and support IE10+
